app/controller/sdl/Abstract/Controller.js

```diff
@@ -660,7 +661,7 @@ SDL.SDLController = Em.Object.extend(
         }
         case 'ScrollableMessage':
         {
-          SDL.ScrollableMessage.deactivate();
+          SDL.ResetTimeoutPopUp.stopRpcProcessing('UI.ScrollableMessage', true);
```

Comment:
Why don't we call deactivate here?

Reply:
@iCollin Beacuse there is no timeout reason in method arguments. `ResetTimeoutPopUp` won't be called from `deactivate` method

Reply:
Thank you, that explains why we call `stopRpcProcessing` but here we also need to call deactivate() so that ScrollableMessage will get a reply when we click steal focus.

Reply:
```suggestion
          SDL.ScrollableMessage.deactivate();
```

It has argument ABORTED which means timeout. Here we need to reply to ScrollableMessage.

Reply:
Fixed in  ccdec68de9a6be27d491716d7e87e411a7139544